{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -16,3 +16,19 @@ universal = 1
 
 [flake8]
 exclude = docs
+
+[aliases]
+{%- if cookiecutter.use_pytest == 'y' %}
+test = pytest
+{%- endif %}
+# Define setup.py command aliases here
+{%- if cookiecutter.use_pytest == 'y' %}
+
+[pytest]
+# Set permanent options for the "pytest" setup command here
```

Comment:
I see, thank you very much for the explanation and pointers! :+1: 
So, considering that 1) both sections are empty and that 2) template users can add a `pytest.ini` if they want to add pytest options, what if we simply leave out these and let the template users decide what to do?